data/human/deep missions.txt

```diff
@@ -2431,3 +2431,340 @@ event "battle in arneb end"
+			`	She smiles broadly at Thorleif and nods.`
+			choice
+				`	(You sigh and shake your head.)`
+			label accompany
+			`	She immediately abandons her kiosk and shrugs, "I didn't care much for that job anyways."`
+			`	You and Thorleif follow her to a local transit hovercraft, where she and Thorleif quickly sit together and you find yourself sitting alone.`
+			`	For the next three hours it sounds as if they are in deep conversation, while you drift in and out of sleep watching as oil fields give way to rainforests and then rainforests shrink before great and arid mountains.`
+			`	At some point you feel Thorleif shaking your shoulder. "We're here."`
+			choice
+				`	(Yawning you rise from your warm seat)`
+			label here
+			`	The town of Grimssalr is nestled along the foothills of the towering Ettin mountains in western Alfheim. Town, is perhaps too generous a word, it is instead more like a couple dozen houses mostly filled with old people long abandoned by their children seeking wealth and adventure elsewhere.`
+			`	The woman, who you've learnt is named Ragnhild, introduces Thorleif to some of the folks here and before long they have found ways to connect their ancient genealogies together and everyone is suddenly treated as family. You eat their meager fare, swap stories, and listen to legends about the old times.`
+			`	As the sun begins to set Thorleif approaches you, "It is time," he says. "The journey to the cairn is only a short hike from here. I would that you accompany me. I think Skaldgar would have wanted that."`	
+			choice
+				`	(Leave Grimssalr with Thorleif.)`
+					goto grimfollow
+			label grimfollow
+			`	You follow Thorleif up a mostly overgrown trail for about 30 minutes before finally stepping out into a wide open plain. The wind is cool and refreshing and the grass blows with a steadiness that is somehow reassuring. There is something special, timeless, about this sudden meadow.`
+			`	You notice over two dozen columns of lain stone arranged in a circle around a much more massive and impressive structure. The lain stone columns must be cairns of various nearby families, but the massive structure in the center is something completely different. The structure is a wall, cut from some sort of metal you don't recognize and filled with letters and images that are unfamiliar to you.`
+			`	"That is an Elf-Stone. It is why we built our Family-Cairns here. Such stones can be found on most worlds in The Deep. Indestructible, and left by the Alfar for some unknowable reason. Some, like this one, have been mostly forgotten. Others though, are important places of celebration and even worship.`
+			`	He points at a smallish cairn next to the Elf-Stone. "This, though, is what we're here for."`
+			choice
+				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`
```

Comment:
As before, my goal here is to keep the text on the screen from becoming too overwhelming. I fear blocks of inscrutable texts. Options like this allow me to keep the text block slightly smaller, and break it up with actions. That, at least, is my reasoning for it.

Reply:
This would be a spot where I don't think a thematic break is really needed. There's no scene transition, right? And this is essentially the climax of the mission chain, so if there's a spot to have a hefty chunk of text this would be it. That's my take on it, but you do you.

Reply:
Alright, sounds good. I will take your suggestion.